src/editor-settings.ts

```diff
@@ -370,15 +370,27 @@ function addEditorSettings (toggleButton: HTMLElement, editor: HTMLElement, prog
 
 	container.style.display = "none";
 
-	toggleButton.addEventListener("click", function () {
+	// This allows you to exit the editor settings by clicking the document also
+	toggleButton.addEventListener("click", function (e) {
```

Comment:
I prefer `evt` or `event` rather than just `e` (to distinguish from "error" which is also commonly abbreviated `e`), but that's a preference thing.